helpers.go

```diff
@@ -369,6 +370,12 @@ func InstallPostSymlinks(post *PostInstall, rootfsPath string) llb.RunOption {
 }
 
 func (s *Spec) GetSigner(targetKey string) (*PackageSigner, bool) {
+	if skipVal, ok := s.Args["DALEC_SKIP_SIGNING"]; ok {
```

Comment:
This is a field in the spec, I don't think we are looking at people adding this to the spec itself, just a built-in arg that people can specify at build time that we'll look for.

See https://github.com/Azure/dalec/blob/577b9ef6845074e178268dc25cd19c4be7334ffa/load.go#L16-L27 for adding this in to the list of known args so the build doesn't error out.

We'll need to check the for the option in the build args on the client.

Reply:
Good catch. It's already been added to the list of known args

Reply:
Fixed this. It's checking the client now.